src/options.rs

```diff
@@ -107,6 +124,7 @@ impl Default for JsOptions {
       fit_to: usvg::FitTo::Original,
       background: None,
       crop: JsCropOptions::default(),
+      log_level: log::LevelFilter::Debug
```

Comment:
It would be better to enable only `Error` by default

Reply:
yep, that makes more sense